chore(release): 0.64.1-rhobs3

sthaha opened the pull request:
This release includes olm bundle

vimalk78 commented on rhobs/olm/subscription/subscription.yaml:
```diff
@@ -0,0 +1,12 @@
+  name: rhobs-prometheus-operator
+  source: rhobs-prometheus-operator
+  sourceNamespace: openshift-marketplace
+  startingCSV: rhobs-prometheus-operator.v0.64.0-rhobs1
```
should this be `v0.64.1-rhobs3` ?

sthaha replied:
We should change this when there is a catalog

vimalk78 commented on rhobs/olm/subscription/catalog-src.yaml:
```diff
@@ -0,0 +1,19 @@
+  icon:
+    base64data: ""
+    mediatype: ""
+  image: quay.io/sthaha/rhobs-prometheus-operator-catalog:latest
```
change to `quay.io/rhobs`

sthaha replied:
+1 .. done

vimalk78 commented on rhobs/olm/manifests/operator/kustomization.yaml:
```diff
@@ -0,0 +1,70 @@
+          spec:
+            containers:
+              - name: prometheus-operator
+                image: quay.io/rhobs/obo-prometheus-operator:v0.64.0-rhobs1
```
`0.64.1-rhobs3`

sthaha replied:
I have added changing this as part of `make_olm_bundle`

vimalk78 commented on rhobs/olm/manifests/csv/rhobs-prometheus-operator.clusterserviceversion.yaml:
```diff
@@ -0,0 +1,54 @@
+  annotations:
+    alm-examples: '[]'
+    capabilities: Basic Install
+  name: rhobs-prometheus-operator.v0.0.0
```
update version

sthaha replied:
I think this is fine since what's in the bundle is what matters.

Commit:
git: merge rhobs-scripts

Commit:
chore(release): v0.64.1-rhobs3

NOTE: This commit was auto-generated by
running rhobs/make-release-commit.sh script

Signed-off-by: Sunil Thaha <sthaha@redhat.com>

vimalk78 commented on rhobs/olm/README.md:
````diff
@@ -0,0 +1,39 @@
+│
+└── subscription # contains olm subscription and catalog-source yaml
+```
+
````
it would be good to add the sequence of commands in the README

vimalk78 reviewed (approved):
/lgtm
